apcd-cms/src/apps/admin_regis_table/views.py

```diff
@@ -56,11 +56,22 @@ def _set_entities(reg_ent):
                 'fein': reg_ent[8]
             }
         def _set_contacts(reg_cont):
+
+            def format_phone_number(num):
+                formatted_num_list = list(reversed(num))
+                for n in range(len(formatted_num_list)):
+                    if n+1 == 4 or n+1 == 8:
+                        formatted_num_list.insert(n+1,'-')
+                    if n+1 == 12 and len(num) > 10:
```

Comment:
Same readability suggestion, same invitation to ignore me
```suggestion
                    if n == 11 and len(num) > 10:
```

Reply:
[Same follow-up suggestion as the other.](https://github.com/TACC/Core-CMS-Custom/pull/52#discussion_r1028540039)